recipe/meta.yaml

```diff
@@ -12,34 +12,35 @@ source:
 
 build:
   number: 0
-  skip: True  # [py<39]
+  skip: True  # [py<310]
   script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation -vv
 
 requirements:
   host:
     - python
     - pip
-    - setuptools
-    - setuptools_scm
+    - setuptools >=59
     - wheel
+    - setuptools-scm >=6.4
   run:
     - python
-    - matplotlib-base >=3.7.0
-    - pandas >=2.1.0,<3.0a0
-    - mizani >=0.11.0,<0.12.0
-    - numpy >=1.23.0
-    - scipy >=1.7.0
+    - matplotlib-base >=3.8.0
+    - pandas >=2.2.0
+    - mizani >=0.13.0,<0.14.dev0
```

Comment:
Shouldn't it be `mizani >=0.13.0,<0.14.0.dev0`. Their releases ended with `0` in x.y.0